pinot-servicemanager/build.gradle.kts

```diff
@@ -0,0 +1,11 @@
+plugins {
+  id("org.hypertrace.docker-publish-plugin")
+}
+
+hypertraceDocker {
+  defaultImage {
+    imageName.set("pinot")
+    tags.forEach { it.onlyIf { false } }
+    tag("servicemanager")
```

Comment:
`hypertrace/pinot:servicemanager` ? That would result in the image having no versioning and sharing a name with an image built completely differently. Would it make more sense to give it its own name, similar to the kafka-zookeeper image?

Reply:
done.